_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
```

Comment:
Yeah, we didn't research it thoroughly enough when we started this project. Sorry :(